source/_patterns/03-layouts/regions/breadcrumb/breadcrumb.twig

```diff
@@ -1,15 +1,17 @@
 <div class="l-breadcrumb">
-  {# l-constrain commented out to accomodate full-bleed default breadcrumb
-    component. Add back if necessary, but avoid nesting l-constrains if possible. #}
-  {# <div class="l-constrain"> #}
-    {% block content %}
-      {% include '@components/breadcrumb/breadcrumb.twig' with {
-        'breadcrumb': [
-          { 'url': '#', 'text': 'Home' },
-          { 'url': '#', 'text': 'About' },
-          { 'text': 'Current Page' }
-        ]
-      } %}
-    {% endblock %}
-  {# </div> #}
+  {% if not unconstrained %}
```

Comment:
This seems confusing checking for the double negative. Can we just check for if `constrain` or if `has-constrain`

Reply:
I did that b/c I wanted the default to have a constrain without having to specify 'constrain: true'... but I see what you mean.  I'll make a change to remove the double negative.